src/main/java/io/cdap/plugin/dlp/SensitiveRecordRedaction.java

```diff
@@ -141,7 +148,7 @@ public void prepareRun(StageSubmitterContext context) throws Exception {
       for (String field : transformationConfig.getFields()) {
         String filterName = String.join(", ", transformationConfig.getFilters())
           .replace("NONE",
-                   String.format("Custom Template (%s)", config.templateId));
+            String.format("Custom Template (%s)", config.templateId));
```

Comment:
Wrong alignment. The previous one is correct. BTW, I think putting on the same line is easier to read.